silver/tests/spec/util/api_get_assert.py

```diff
@@ -12,11 +12,12 @@ def setUp(self):
         self.client.force_authenticate(user=admin_user)
 
     def assert_get_data(self, url, expected_data):
+        many = isinstance(expected_data, list)
         response = self.client.get(url, format='json')
         request = response.wsgi_request
         expected = self.serializer_class(expected_data,
```

Comment:
I'll create a serializer encoding test to make sure it doesn't change without our knowledge.